nemo/collections/asr/models/hybrid_rnnt_ctc_models.py

```diff
@@ -601,6 +601,9 @@ def validation_pass(self, batch, batch_idx, dataloader_idx):
         if AccessMixin.is_access_enabled():
             AccessMixin.reset_registry(self)
 
+        # adding this as return values are no longer logger automatically in PTL2.0
```

Comment:
remove the comments

Reply:
Fixed in latest commit. Also added necessary logging changes for label_models, slu_models, ssl_models

Reply:
Thanks Kunal. @athitten how is this different from adding to `test_step_outputs` or `validation_step_outputs` ?